src/js_realm.hpp

```diff
@@ -220,6 +223,7 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
         {"deleteAll", wrap<delete_all>},
         {"write", wrap<write>},
         {"addListener", wrap<add_listener>},
+        {"wait", wrap<wait_for_download_completion>},
```

Comment:
yes this is public. I just kept in sync to the other methods. We can shape it better I guess. 
My reasoning here is user is not aware what wait_for_download means. So generic wait method after open seemed better. like Realm.open(config).wait().
And also wait_for_download does not make sense in local realms but I made it possible to use wait for that as well. 
If this is exposed as onRealmSynchronized it may leave the user with impression it can be called multiple times during the app lifetime. And still the name don't make sense when opening local realms.